mappings/net/minecraft/client/network/ServerAddressResolver.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_6369 net/minecraft/client/network/ServerAddressResolver
+	FIELD field_33743 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_33744 tryResolve Lnet/minecraft/class_6369;
```

Comment:
```suggestion
	FIELD field_33744 DEFAULT Lnet/minecraft/class_6369;
```
This is a default instance of the address resolver resolving the inet address directly.